tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -954,10 +955,16 @@ constexpr char const *kBlobClassName = "RBlob";
 /// The class name of the RNTuple anchor
 constexpr char const *kNTupleClassName = "ROOT::Experimental::RNTuple";
 
+} // anonymous namespace
+
 /// The RKeyBlob writes an invisible key into a TFile.  That is, a key that is not indexed in the list of keys,
 /// like a TBasket.
+/// NOTE: out of anonymous namespace because otherwise ClassDefInline fails to compile
+/// on some platforms.
 class RKeyBlob : public TKey {
```

Comment:
This becomes now a global symbol name, doesn't it? In any case, should we put it in `ROOT::Experimental::Internal`?

Reply:
Yes and yes :)

An anonymous namespace is usually implemented by an untypable name .... which by definition wont be type-able by the `ClassDef` macro or dictionary generation.